webpack.proxyPortal.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * © 2014 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: LGPL-3.0-or-later
+ */
+
+/* eslint-env node */
+
+const path = require('path');
+
+module.exports = () => ({
+	mode: 'development',
+	entry: path.resolve(__dirname, 'src/loader/bootstrap.js'),
+	output: {
+		path: path.resolve(__dirname, 'build/proxyPortal'),
+		filename: 'loader.js',
+	},
+	devtool: 'inline-source-map',
+	module: {
+		rules: [
+			{
+				test: /\.js$/,
+				exclude: /node_modules/,
+				use: [
+					{
+						loader: 'babel-loader',
+						options: {},
+					},
+				],
+			},
+		],
+	},
+	devServer: {
+		port: 9090,
```

Comment:
This is the same port as the themes watcher uses. I wonder if we should be using the "portfinder" package to make sure we always get hold of *some* port even if the one we ask for isn't available.

Reply:
In this case this doesn't pose any problem because if you are using this you are doing a very specific task and chances are that the port is available.

Also, using a free port means that the number will vary and it's easier to just remember that we usually use 9090.